### ISSUE TITLE
Why does is the HTML not 'active' (jQueryUI, et. al)?

### ISSUE THREAD

**Issue by infracritical** [vividvilla/csvtotable#10]
OK, I was able to use the workaround, then also did the upgrade; in both circumstances, I was able to generate HTML.  However...upon taking closer look at the actual output file, it looks like it uses "jQueryUI" and another JavaScript-embedded script.  It should be actively displaying the information (sorting, searching, etc.)

I was unable to see this done.  What am I doing wrong?

**Comment by infracritical** [comment]
I have an example up and running: **http://unixworks.net/test921/i3.html**.

**Comment by vividvilla** [comment]
Can you please paste or upload sample CSV format here? Seems like you didn't specify proper `quotechar` or `delimiter`.

**Comment by infracritical** [comment]
I found the problem.  There were several 'ghost characters' in the converted CSV file, despite the fact that I had used "iconv" to format the CSV file as UTF-8.  I created a separate sample, and it appears to be working.

The sample is up and running: **http://unixworks.net/test921/t.html**.

I will go ahead and close this issue; I hope that I wasn't too much of a PITA, but figured that if I ask here, it would help others who may encountered something similar.

Thanks again!
